### ISSUE TITLE
MacOS brew problems

### ISSUE THREAD

**Issue by storopoli** [phiresky/ripgrep-all#83]
Installed with `brew install rga`

Maybe related with #80 

```bash
adapter: decompress
Error: invalid gzip header

--- preprocessor command failed: '"/usr/local/bin/rga-preproc"
```

And also
```bash
adapter: pandoc
[WARNING] Deprecated: --atx-headers. Use --markdown-headings=atx instead.
```

**Comment by opennota** [comment]
They have removed the deprecated option, so now it's failing with

```
Unknown option --atx-headers.
Try pandoc --help for more information.
```

It's not MacOS-specific (I'm on Gentoo).

```
$ pandoc --version
pandoc 3.0.1
```

**Comment by DziubanMaciej** [comment]
Also visible on Windows.

**Comment by tddschn** [comment]
Is there a workaround?

**Comment by DziubanMaciej** [comment]
> Is there a workaround?

I downloaded older version of pandoc (before 3.0 I think) and replaced the executable. It works, although it might be secretly breaking some other programs...

**Comment by phiresky** [comment]
Fixed with 0.10
